client/tinychain/ml/__init__.py

```diff
@@ -98,3 +99,11 @@ def create(cls, *args, **kwargs):
     @abstractmethod
     def load(cls, *args, **kwargs):
         pass
+
+
+class Parameter:
+
```

Comment:
I think it would be better to use `Tensor` here, or just not specify a type (that way this will still work if in the future someone decides to use a `Sparse` tensor)